modules/unsupported/wfs-ng/src/main/java/org/geotools/data/wfs/WFSFeatureSource.java

```diff
@@ -133,6 +133,30 @@ public WFSDataStore getDataStore() {
         return (WFSDataStore) super.getDataStore();
     }
 
+    /**
+     * This method changes the query object so that all propertyName references are resolved to
+     * simple attribute names against the schema of the feature source.
+     *
+     * <p>For example, this method ensures that propertyName's such as "gml:name" are rewritten as
+     * simply "name".
+     */
+    @Override
+    protected Query resolvePropertyNames(Query query) {
+        // Resolving is not conform with the specification of a wfs propertyname which is a QName
+        // from w3.org, see
```

Comment:
No need to wrap comment on the next line, can you use just 3 lines instead of 2?